qiskit/optimization/problems/__init__.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019, 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+=================================================================
+Optimization stack for Aqua (:mod:`qiskit.optimization.problems`)
+=================================================================
+
+.. currentmodule:: qiskit.optimization.problems
+
+Structures for defining an optimization problem and its solution
+==========
+
+.. autosummary::
+   :toctree: ../stubs/
+   :nosignatures:
+
+   Constraint
+   LinearExpression
+   LinearConstraint
+   QuadraticExpression
+   QuadraticConstraint
+   QuadraticObjective
+   QuadraticProgram
+   Variable
+
+N.B. Additional classes Constraint, LinearConstraint, QuadraticConstraint,
```

Comment:
One thought is that instead of having a single list here have 2 of them where the noted classes are in the second list and have the above comment about the way they come about apply to the list